Gemfile

```diff
@@ -26,7 +26,6 @@ gem 'stimulus-rails'
 
 # Build JSON APIs with ease [https://github.com/rails/jbuilder]
 gem 'jbuilder'
-gem 'rails-controller-testing'
 # Use Redis adapter to run Action Cable in production
 # gem "redis", "~> 4.0"
 
```

Comment:
- [ ] Please use the   [getting started link](https://github.com/heartcombo/devise) to setup Devise, the first step from this [guide](https://github.com/heartcombo/devise#getting-started) is to run `bundle add devise`